landsat_localindex/db/queries.go

```diff
@@ -7,14 +7,18 @@ import (
 	"github.com/venicegeo/geojson-go/geojson"
 )
 
+// GetSceneByID looks up a single scene by its product ID
 func GetSceneByID(tx *sql.Tx, productID string) (*LandsatLocalIndexScene, error) {
-	var boundsBytes []byte
+	var wrsBoundsBytes []byte
+	var mtlBoundsBytes []byte
 	scene := LandsatLocalIndexScene{}
 
 	rows, err := tx.Query(`
-		SELECT product_id, acquisition_date, cloud_cover, scene_url, ST_AsGeoJSON(bounds)
+		SELECT product_id, acquisition_date, cloud_cover, scene_url, ST_AsGeoJSON(bounds), 
+		       ST_AsGeoJSON(ST_MakePolygon(ST_MakeLine(ARRAY[corner_ul, corner_ur, corner_lr, corner_ll, corner_ul])))
 		FROM public.scenes
 		WHERE product_id=$1
+			AND corner_ll IS NOT NULL 
```

Comment:
We're just assuming the others are good if this one is?

Reply:
That is more a sanity check to see that corner info is set at all. Since there is a time gap between the ingest and the corner info fetch, we don't want to fall to the trap of using incomplete data.